templates/security/index.html

```diff
@@ -496,17 +596,88 @@ <h2>Ubuntu security disclosure policy</h2>
+        <div class="row">
+          <div class="col-2">
+            <h3 class="p-heading--5">Join the discussion</h3>
+          </div>
+          <div class="col-4">
+            <ul class="p-list--divided">
+              <li class="p-list__item">
+                <a href="https://lists.ubuntu.com/mailman/listinfo/ubuntu-hardened"
+                   onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'Ubuntu security updates mailing list', 'eventLabel' : 'Join the discussion', 'eventValue' : undefined });">Ubuntu security updates mailing list&nbsp;&rsaquo;</a>
+              </li>
+              <li class="p-list__item">
+                <a href="https://lists.ubuntu.com/mailman/listinfo/ubuntu-security-announce"
+                   onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'Security announcements mailing list', 'eventLabel' : 'Join the discussion', 'eventValue' : undefined });">Security announcements mailing list&nbsp;&rsaquo;</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-2">
+            <h3 class="p-heading--5">Canonical is offering Expanded Security Maintenance</h3>
+          </div>
+          <div class="col-4">
+            <p>Canonical is offering Ubuntu Expanded Security Maintenance (ESM) for security fixes and essential packages.</p>
+            <div class="p-cta-block">
+              <a href="/support/esm">Find out more about ESM&nbsp;&rsaquo;</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
+        <h2>
+          <a href="/blog/tag/security">Latest from&nbsp;our&nbsp;blog&nbsp;&rsaquo;</a>
+        </h2>
+      </div>
+      <div class="p-section--shallow">
+        <div class="row" id="ubuntu-desktop-latest"></div>
+      </div>
+    </div>
+
+    <template id="ubuntu-desktop-template">
+      <div class="col-3 col-medium-2 u-equal-height-items">
+        <div class="u-crop--16-9">
+          <div class="article-image p-image-wrapper"></div>
+        </div>
+        <h3 class="p-heading--5">
+          <a class="article-link article-title"></a>
+        </h3>
+        <p class="article-excerpt"></p>
+      </div>
+    </template>
+  </section>
+
+  {% include "/shared/forms/form-template.html" %}
+
+  <script src="{{ versioned_static('js/dist/latest-news.js') }}"></script>
+  {# djlint:off #}
+  <script>  
+    canonicalLatestNews.fetchLatestNews(
+      {
+        articlesContainerSelector: "#ubuntu-desktop-latest",
+        articleTemplateSelector: "#ubuntu-desktop-template",
+        excerptLength: 200,
+        tagId: 1364,
+        linkImage: true,
+        limit: 4
+      }
+    )
+</script>
```

Comment:
This is fairly common, could we try using/making a shared template for this?

Reply:
I normally would, but this seems to be the only page in this bubble that is using it. We have slightly different versions of it in different bubbles, and I agree we should align them all into one partial that serves all purposes like the one we used to have, but that task does not seem in scope for this pr

Reply:
Makes sense